Application/Use Cases/CommandHandlers/LocationCommandHandlers/BulkInsertLocationCommandHandler.cs

```diff
@@ -0,0 +1,50 @@
+using Application.Use_Cases.Commands.LocationCommands;
+using Domain.Entities;
+using Domain.Errors;
+using Domain.Repositories;
+using Domain.Utils;
+using MediatR;
+
+namespace Application.Use_Cases.CommandHandlers.LocationCommandHandlers;
+
+public class BulkInsertLocationCommandHandler : IRequestHandler<BulkInsertLocationCommand, Result<Unit>>
+{
+    private readonly ILocationRepository _repository;
+
+    public BulkInsertLocationCommandHandler(ILocationRepository repository)
+    {
+        _repository = repository;
+    }
+    
+    public async Task<Result<Unit>> Handle(BulkInsertLocationCommand request, CancellationToken cancellationToken)
+    {
+        var locations = CreateLocations(request.MaxFloorNo, request.RoomsPerFloor);
+        foreach (var location in locations)
+        {
+            var resultObj = await _repository.AddAsync(location);
+            if (!resultObj.IsSuccess)
+            {
+                return Result<Unit>.Failure(EntityErrors.CreateFailed(nameof(Location), resultObj.Error!.Description ?? "Something went wrong when bulk inserting locations. You most likely inserted values that were already present in the database."));
+            }
+        }
+        return Result<Unit>.Success(default);
+    }
+    
+    private ICollection<Location> CreateLocations(int maxFloorNo, int roomsPerFloor)
+    {
+        var locations = new List<Location>();
+        for (var floorIndex = 0; floorIndex < maxFloorNo; floorIndex++)
+        {
+            for (var roomIndex = 1 + 25*floorIndex; roomIndex <= roomsPerFloor + 25*floorIndex; roomIndex++)
```

Comment:
Aici as sugera sa punem o constanta ca sa se inteleaga ce inseamna 25. Nu ar trebui sa punem magic numbers in cod